### ISSUE TITLE
[Issue]:  HDHomeRun Dual Live tv failure:  MediaBrowser.Controller.LiveTv.LiveTvConflictException

### ISSUE THREAD

**Issue by psyciknz** [jellyfin/jellyfin#7729]
### Please describe your bug

Attempting to watch Live tv (or set a recording).    Once trying to start a live tv stream the MediaBrowser.Controller.LiveTv.LiveTvConflictException error is thrown, with no explanation as to what it means.  Or what there is a conflict with (if the exception is anything to go by).

I found some posts about using the hdhomerun dual in legacy mode with the windows software, which is running, and firmware updated (for reference, after some tweaking, plex is now working).   I did just try stopping plex, and just having jellyfin running on the thought that they were both holding a connection, but that had no effect.

I don't have the hdhomerun setup with in in emby mode for the bda drivers - but had tested that last night to no positive effect.

### Jellyfin Version

10.8.0 (and 10.7.7 stable) - upgraded after seeing issues

### Environment

```markdown
- OS: windows 10
- Virtualization: n/a
- Clients: n/a
- Browser: Edge
- FFmpeg Version:  Found ffmpeg version 5.0.1 (according to jellyfin log)
- Playback Method: browser
- Hardware Acceleration: unsure, I assume yes, but not relevant at this point I think
- Plugins: slack, trakt
- Reverse Proxy: n/a
- Base URL: http://localhost:8096
- Networking: Server is in 192.160.10 subnet, hdhomerun in 192.168.40.   But discovery doesn't seem to be an issue.
- Storage:
```

### Jellyfin logs

```shell
[07:09:17] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Found ffmpeg version 5.0.1
[07:09:17] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available decoders: ["libdav1d", "av1", "av1_cuvid", "av1_qsv", "h264", "h264_qsv", "h264_cuvid", "hevc", "hevc_qsv", "hevc_cuvid", "mpeg2video", "mpeg2_qsv", "mpeg2_cuvid", "mpeg4", "mpeg4_cuvid", "msmpeg4", "vc1_qsv", "vc1_cuvid", "vp8", "libvpx", "vp8_cuvid", "vp8_qsv", "vp9", "libvpx-vp9", "vp9_cuvid", "vp9_qsv", "aac", "ac3", "flac", "mp3"]
[07:09:17] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available encoders: ["libx264", "h264_amf", "h264_nvenc", "h264_qsv", "libx265", "hevc_amf", "hevc_nvenc", "hevc_qsv", "mpeg4", "msmpeg4", "libvpx", "libvpx-vp9", "aac", "libfdk_aac", "ac3", "flac", "libmp3lame", "libopus", "libvorbis", "srt"]
[07:09:17] [WRN] [13] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://127.0.0.1:8096/system/info/public to 127.0.0.1 in 0:00:00.5599615 with Status Code 503
[07:09:17] [WRN] [24] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://localhost:8096/ScheduledTasks?IsEnabled=true to 127.0.0.1 in 0:00:00.5595924 with Status Code 503
[07:09:17] [WRN] [21] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://localhost:8096/system/info/public to 127.0.0.1 in 0:00:00.5596002 with Status Code 503
[07:09:17] [WRN] [26] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://localhost:8096/system/info/public to 127.0.0.1 in 0:00:00.559601 with Status Code 503
[07:09:17] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available filters: ["deinterlace_qsv", "hwupload_cuda", "hwupload_vaapi", "overlay_opencl", "overlay_qsv", "overlay_cuda", "scale_cuda", "scale_opencl", "scale_qsv", "tonemap_cuda", "tonemap_opencl", "vpp_qsv", "yadif_cuda", "zscale", "alphasrc"]
[07:09:17] [WRN] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Filter: overlay_vaapi with option Action to take when encountering EOF from secondary input is not available
[07:09:17] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available hwaccel types: ["cuda", "dxva2", "qsv", "d3d11va", "opencl"]
[07:09:17] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: C:\Program Files\Jellyfin\Server\ffmpeg.exe
[07:09:17] [INF] [1] Emby.Server.Implementations.ApplicationHost: ServerId: ae4cec6b6d2b42a8be1dd98cec5ac4b6
[07:09:18] [INF] [1] Emby.Server.Implementations.ApplicationHost: Executed all pre-startup entry points in 0:00:00.3671641
[07:09:18] [INF] [1] Emby.Server.Implementations.ApplicationHost: Core startup complete
[07:09:18] [INF] [1] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Loading live tv data from M:\JellyFin\data\livetv\timers.json
[07:09:18] [INF] [10] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory P:\ServerFolders\Music
[07:09:18] [INF] [25] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory P:\ServerFolders\Recorded TV
[07:09:18] [INF] [21] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory P:\ServerFolders\Recordings
[07:09:18] [INF] [24] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory P:\ServerFolders\Videos
[07:09:18] [INF] [24] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory P:\ServerFolders\Videos_To_check\podsync
[07:09:18] [INF] [1] Emby.Server.Implementations.ApplicationHost: Executed all post-startup entry points in 0:00:00.3404566
[07:09:18] [INF] [1] Main: Startup complete 0:00:06.4402418
[07:09:19] [INF] [19] Emby.Server.Implementations.ScheduledTasks.TaskManager: StartupTrigger fired for task: Update Plugins
[07:09:19] [INF] [19] Emby.Server.Implementations.ScheduledTasks.TaskManager: Queuing task PluginUpdateTask
[07:09:19] [INF] [24] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing Update Plugins
[07:09:23] [INF] [30] Emby.Server.Implementations.ScheduledTasks.TaskManager: Update Plugins Completed after 0 minute(s) and 4 seconds
[07:09:23] [INF] [30] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
[07:09:35] [INF] [21] Emby.Server.Implementations.HttpServer.WebSocketManager: WS 127.0.0.1 request
[07:10:00] [WRN] [37] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://127.0.0.1:8096/Items/5ee0772a1e00aafd3efe49a6633c7a15/Images/Primary?fillHeight=279&fillWidth=186&quality=96&tag=0342a11bd40c47cb7f19254ae0f70750 to 127.0.0.1 in 0:00:00.6175762 with Status Code 200
[07:10:00] [WRN] [37] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://127.0.0.1:8096/Items/2c6c13bc228ac30a5dd62c22ed0cf358/Images/Primary?fillHeight=279&fillWidth=186&quality=96&tag=61e904b3f1a50e5f86853df2bd7d8ac8 to 127.0.0.1 in 0:00:00.6686264 with Status Code 200
[07:10:00] [WRN] [44] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://127.0.0.1:8096/Items/20b853b8a136ecc121e5d6922d63152b/Images/Primary?fillHeight=279&fillWidth=186&quality=96&tag=0aa799b3a164a6321d3f5b8433e7149f to 127.0.0.1 in 0:00:00.6764578 with Status Code 200
[07:10:16] [INF] [38] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Loading live tv data from M:\JellyFin\data\livetv\seriestimers.json
[07:10:17] [INF] [30] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for David. EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[07:10:17] [INF] [30] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile=Anonymous Profile, Path=http://192.168.40.8, AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=Transcode, TranscodeReason=VideoCodecNotSupported ) media:/videos/ae38414d-6971-ae82-a5e3-27a0e933c48d/master.m3u8?MediaSourceId=native_3296d406dcc9cb9b62aeaef99612ba6b_6dbc8630f6b33302f72643edd40a6a86&VideoCodec=h264&AudioCodec=aac,mp3,mp2&AudioStreamIndex=-1&VideoBitrate=139808000&AudioBitrate=192000&MaxWidth=1680&api_key=<token>&TranscodingMaxAudioChannels=2&RequireAvc=false&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&h264-profile=high,main,baseline,constrainedbaseline,high10&h264-level=52&h264-deinterlace=true&TranscodeReasons=VideoCodecNotSupported
[07:10:17] [INF] [30] Emby.Server.Implementations.LiveTv.LiveTvManager: Opening channel stream from Emby, external channel Id: hdhr_1
[07:10:17] [INF] [30] Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV: Streaming Channel hdhr_1
[07:10:17] [INF] [30] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: GetChannelStream: channel id: hdhr_1. stream id: native_3296d406dcc9cb9b62aeaef99612ba6b_6dbc8630f6b33302f72643edd40a6a86 profile: native
[07:10:17] [INF] [30] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Opening HDHR UDP Live stream from 192.168.40.8
[07:10:17] [ERR] [27] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening live stream:
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
[07:10:17] [ERR] [27] Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunHost: Error opening tuner
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunManager.StartStreaming(IPAddress remoteIp, IPAddress localIp, Int32 localPort, IHdHomerunChannelCommands commands, Int32 numTuners, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.HdHomerun.HdHomerunUdpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
[07:10:17] [ERR] [27] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request. URL POST /Items/ae38414d6971ae82a5e327a0e933c48d/PlaybackInfo.
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelStreamWithDirectStreamProvider(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.GetChannelStream(String id, String mediaSourceId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvMediaSourceProvider.OpenMediaSource(String openToken, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStreamInternal(LiveStreamRequest request, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStream(LiveStreamRequest request, CancellationToken cancellationToken)
   at Jellyfin.Api.Helpers.MediaInfoHelper.OpenMediaSource(HttpRequest httpRequest, LiveStreamRequest request)
   at Jellyfin.Api.Controllers.MediaInfoController.GetPostedPlaybackInfo(Guid itemId, Nullable`1 userId, Nullable`1 maxStreamingBitrate, Nullable`1 startTimeTicks, Nullable`1 audioStreamIndex, Nullable`1 subtitleStreamIndex, Nullable`1 maxAudioChannels, String mediaSourceId, String liveStreamId, Nullable`1 autoOpenLiveStream, Nullable`1 enableDirectPlay, Nullable`1 enableDirectStream, Nullable`1 enableTranscoding, Nullable`1 allowVideoStreamCopy, Nullable`1 allowAudioStreamCopy, PlaybackInfoDto playbackInfoDto)
   at lambda_method1013(Closure , Object )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Jellyfin.Server.Middleware.ServerStartupMessageMiddleware.Invoke(HttpContext httpContext, IServerApplicationHost serverApplicationHost, ILocalizationManager localizationManager)
   at Jellyfin.Server.Middleware.WebSocketHandlerMiddleware.Invoke(HttpContext httpContext, IWebSocketManager webSocketManager)
   at Jellyfin.Server.Middleware.IpBasedAccessValidationMiddleware.Invoke(HttpContext httpContext, INetworkManager networkManager)
   at Jellyfin.Server.Middleware.LanFilteringMiddleware.Invoke(HttpContext httpContext, INetworkManager networkManager, IServerConfigurationManager serverConfigurationManager)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.QueryStringDecodingMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.ReDoc.ReDocMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.RobotsRedirectionMiddleware.Invoke(HttpContext httpContext)
   at Jellyfin.Server.Middleware.LegacyEmbyRouteRewriteMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.InvokeCore(HttpContext context)
   at Jellyfin.Server.Middleware.ResponseTimeMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.ExceptionMiddleware.Invoke(HttpContext context)
[07:10:17] [WRN] [27] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://127.0.0.1:8096/Items/ae38414d6971ae82a5e327a0e933c48d/PlaybackInfo?UserId=a0f4ebed18a24c7594273c27b7cefba5&StartTimeTicks=0&IsPlayback=true&AutoOpenLiveStream=true&MaxStreamingBitrate=140000000 to 127.0.0.1 in 0:00:01.0773548 with Status Code 500
[07:10:23] [INF] [38] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[07:10:35] [INF] [44] Emby.Server.Implementations.Session.SessionWebSocketListener: Lost 1 WebSockets.
```

### FFmpeg logs

```shell
Not created
```

### Please attach any browser or client logs here

Requesting url without automatic networking: http://localhost:8096/Items/ae38414d6971ae82a5e327a0e933c48d/PlaybackInfo?UserId=a0f4ebed18a24c7594273c27b7cefba5&StartTimeTicks=0&IsPlayback=true&AutoOpenLiveStream=true&MaxStreamingBitrate=140000000
main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2 
        
       POST http://localhost:8096/Items/ae38414d6971ae82a5e327a0e933c48d/PlaybackInfo?UserId=a0f4ebed18a24c7594273c27b7cefba5&StartTimeTicks=0&IsPlayback=true&AutoOpenLiveStream=true&MaxStreamingBitrate=140000000 500 (Internal Server Error)
b @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
value @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
value @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
value @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
H @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
le @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
(anonymous) @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
Promise.then (async)
ie @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
(anonymous) @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
Promise.then (async)
(anonymous) @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
Promise.then (async)
(anonymous) @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
Promise.then (async)
K @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
(anonymous) @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
Promise.then (async)
B @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
(anonymous) @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
Promise.then (async)
(anonymous) @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
Promise.then (async)
l.play @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
m @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
g @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
bt @ main.jellyfin.bundle.js?d70fad224dbec6b9e2a3:2
index.html#/livetv.html:1 
        
       Uncaught (in promise) Response {type: 'basic', url: 'http://localhost:8096/Items/ae38414d6971ae82a5e327…OpenLiveStream=true&MaxStreamingBitrate=140000000', redirected: false, status: 500, ok: false, …}

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ruthara** [comment]
I am another user experiencing this bug (unable to open live tv stream with the noted exception).
I suspect the common variable is an older model HDHR duo, though it is otherwise working correctly, has updated firmware, and shows the channel list correctly.
Note that after setting up the HDHR, the config/livetv.xml file listed the tuner count as zero.  Perhaps that means 'unknown', but I have also tried manually setting the count, but the same behavior exists.
It would be helpful if someone can expand on the specific error exception as it has very little detail from the log.

Environment:
Jellyfin 10.8.4 (no plugins)
Docker/Mac/Arm; but also directly under Ubuntu/x64

[jellyfin.log](https://github.com/jellyfin/jellyfin/files/9648409/jellyfin.log)
[livetv.xml.txt](https://github.com/jellyfin/jellyfin/files/9648414/livetv.xml.txt)

**Comment by ruthara** [comment]
Duplicate of #7154

**Comment by psyciknz** [comment]
> I am another user experiencing this bug (unable to open live tv stream with the noted exception). I suspect the common variable is an older model HDHR duo, though it is otherwise working correctly, has updated firmware, and shows the channel list correctly. Note that after setting up the HDHR, the config/livetv.xml file listed the tuner count as zero. Perhaps that means 'unknown'. I have also tried manually setting the count, but the same behavior exists. It would be helpful if someone can expand on the specific error exception as it has very little detail from the log.
> 
> Environment: Jellyfin 10.8.4 (no plugins) Docker/Mac/Arm; but also directly under Ubuntu/x64
> 
> [jellyfin.log](https://github.com/jellyfin/jellyfin/files/9648409/jellyfin.log) [livetv.xml.txt](https://github.com/jellyfin/jellyfin/files/9648414/livetv.xml.txt)

I switched to nextpvr for controlling the tuner (and iptv).  It's much more reliable.  There is a nextpvr/jellyfin plugin  that gives a view of the guide and recordings.   Yes it's another container but it's so reliable (I'd been using nextpvr since before plex was a thing, and it was great then for tuners/live tv)

**Comment by SenorSmartyPants** [comment]
This should be fixed by #8726 . This is available in the nightly branch.
